hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieRealtimeTableCompactor.java

```diff
@@ -126,57 +209,7 @@ public HoodieCommitMetadata compact(JavaSparkContext jsc, HoodieWriteConfig conf
     return metadata;
   }
 
-  private boolean isCompactionSucceeded(HoodieCommitMetadata result) {
```

Comment:
why the shuffling around of code?

Reply:
please trim it down to just the code that changes.. (happy to take another separate PR for rearranging stuff, if thats your intention)

Reply:
1. Many method here, including ExecuteCompaction method has been refactored and most of the stuff moved around and rest of it yes, it's just meant for re-arranging stuff. Will try to keep minimal effect going forward, let me also try to fix this one as much as possible.